go.mod

```diff
@@ -8,14 +8,14 @@ require (
 	github.com/calee0219/fatal v0.0.1
 	github.com/gin-contrib/cors v1.7.0
 	github.com/gin-gonic/gin v1.9.1
-	github.com/omec-project/CommonConsumerTestData v1.1.1
-	github.com/omec-project/amf v1.1.2-0.20240312010902-73424b00e9e9
-	github.com/omec-project/aper v1.1.0
-	github.com/omec-project/http2_util v1.1.0
-	github.com/omec-project/logger_util v1.1.0
-	github.com/omec-project/nas v1.1.5-0.20240220010435-2ab8bd499c0f
-	github.com/omec-project/ngap v1.1.1-0.20240127060744-31784065bc81
-	github.com/omec-project/openapi v1.1.0
+	github.com/omec-project/CommonConsumerTestData v1.2.0
+	github.com/omec-project/amf v1.2.0
```

Comment:
last time you had removed dependency on AMF right?

Reply:
No, I did not finished to do that.